src/components/AppLayout.js

```diff
@@ -153,15 +155,22 @@ const AppLayoutElem = styled.div`
 `;
 
 export function AppLayoutView(props) {
-  const { authMemberDoc, showModal } = props;
-  const headerProps = {
-    memberDetails: authMemberDoc && {
+  const { authFirebaseUser, authMemberDoc, authMemberDocIsLoaded, showModal } = props;
+  let memberDetails = false;
```

Comment:
`memberDetails` here is being overloaded with a bunch of different potential type signatures, either boolean false, an actual `memberDetails`  blob, or a fake one that masquerades as one with a false `profileUrl`. For now without a proper type system I don't think it's worth treating them as separate types, but in the future I'd probably just make `memberDetails` be a union of unambiguously different types so that the usage is obvious and verifiable.